projects/clr/hipamd/src/hip_platform.cpp

```diff
@@ -813,6 +813,11 @@ void PlatformState::init() {
   for (auto& it : statCO_.functions_) {
     it.second->resize_dFunc(g_devices.size());
   }
+  amd::RuntimeTearDown::RegisterTearDownCallback("PlatformState static fatbin cleanup", [this]() {
+    runtime_destroyed_.store(true, std::memory_order_release);
+    hipError_t err = statCO_.removeAllFatBinaries();
+    assert(err == hipSuccess);
```

Comment:
Don't need assert since it's success always, removeAllFatBinaries should be void.